packages/botonic-plugin-contentful/src/cms/cms-error.ts

```diff
@@ -201,29 +209,24 @@ export class ContentfulExceptionWrapper {
     }
     const msg = `Error calling ${this.wrappee}.${method}${content}.`
     const exception = new CmsException(msg, contentfulError)
-    if (this.logErrors) {
-      console.error(exception.toString())
-      this.processError(contentfulError)
-    }
+    const err = this.processError(contentfulError)
+    this.logger(`${msg} Due to ${err}`)
+
     throw exception
   }
 
-  processError(contentfulError: Error) {
-    if (this.logStack) {
-      if (contentfulError instanceof MultiError) {
-        console.error('due to:')
-        for (const e of reduceMultiError(contentfulError)) {
-          this.processError(e)
-        }
-      } else {
-        console.error('due to', contentfulError)
-        if (contentfulError.stack) {
-          console.error(
-            'at',
-            contentfulError.stack.split('\n').slice(0, 3).join('\n')
-          )
-        }
+  processError(contentfulError: Error): string {
+    let err = ''
+    if (contentfulError instanceof MultiError) {
+      err += ' Due to:'
+      for (const e of reduceMultiError(contentfulError)) {
+        err += this.processError(e)
       }
+    } else if (this.logStack && contentfulError.stack) {
+      err += contentfulError.stack.split('\n').slice(0, 5).join('\n')
```

Comment:
Maybe a bit "hacky" Why are you slicing here?

Reply:
we don't want to spam the logs with too long callstack